modules/fbs/includes/fbs.availability.inc

```diff
@@ -253,8 +253,18 @@ function fbs_availability_holdings($provider_ids) {
             $tmp_issues[$key]['total'] += 1;
           }
           else {
+            // Generate "vol key". Both volume and volumeNumber can be empty so
+            // if one is empty just use the other or if both are present we use
+            // "volume-volumeNumber".
+            $separator = '';
+            $volume = $material->periodical->volume;
+            $volume_number = $material->periodical->volumeNumber;
+            if (!empty($volume) && !empty($volume_number)) {
+              $separator = '-';
+            }
```

Comment:
This could be reduced to something like:

```php
$volume_data = [$material->periodical->volume, $material->periodical->volumeNumber];
$volume_string = implode('-', array_filter($volume_data);
```

Reply:
This is much more clever, so I think we should use that ;) I'll update PR.

Reply:
Updated, thanks for input 👍